README.md

```diff
@@ -62,6 +63,38 @@ druid's existing functionality and widgets.
 [![flex.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/flex.png)](./druid/examples/flex.rs)
 [![custom_widget.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/custom_widget.png)](./druid/examples/custom_widget.rs)
 
+## Goals
+
+Druid's primary goal is to provide a smooth and polished user experience with
+pixel perfect design and no behavioral quirks. In order to achieve this we
+strive for a variety of things:
+
+- Make it easy to build and package on all supported platforms.
+- Provide abstractions to avoid platform specific quirks.
+- Encourage good application behavior across platforms.
+- Automate platform resolution independence.
+- Enable robust internationalization.
+- Ensure accessibility to a wide group of people.
+- Produce small and fast binaries with low memory usage.
+- Have a small dependency tree with a high quality code base and good organization.
+- Focus on powerful, desktop-grade applications.
+- Provide the basic set of widgets needed by most applications.
+- Simplify creation of complex, special purpose widgets.
+
+### Non-Goals
+
+In order to fulfill those goals, we cannot support every use case. Luckily
+the Rust community is working on a variety of different libraries with
+different goals, so here are some of druid's non-goals and possible
+alternatives that can offer those capabilities:
+
+- Use the the platform-native widgets or mimic them. ([Relm])
+- Embed easily into custom render pipelines. ([Conrod])
+- Implement a specific architectural style such as Elm. ([Iced], [Relm])
+- Support rendering to HTML when compiling for WASM. ([Iced], [Moxie])
+
+There is also a collection of many more ongoing [Rust-native GUI experiments].
```

Comment:
Does this link work? I don't see it when I skim.

```suggestion
Druid is just one of many ongoing [Rust-native GUI experiments]. If it doesn't suit your use case, perhaps one of the others will!
```

Reply:
The link does work. For reference when viewing the changes on GitHub, you can click on the three dots button in the top right and choose "View file" then that will open that as formatted markdown and you can click on the link.